app/models/course_queue.rb

```diff
@@ -32,7 +32,20 @@ def request(requester:, description:, location:, group:)
   end
 
   def outstanding_requests
-    course_queue_entries.where(resolved_at: nil).order('created_at ASC')
+    if course.sort_by
```

Comment:
Have you considered putting this knob at the course queue level rather than the course level?

Reply:
I don't know if it makes more sense at the queue or course level. On one hand, at the course level means all instructors have to stick to the same course policy. On the other hand, the flexibility at the queue level might be nice. I am really not sure. I think course level is "safer" as it is more strict.

Reply:
Cool, sounds good to me. We can start with it here and always move it around if need be.